crates/astria-sequencer/src/bridge/query.rs

```diff
@@ -15,6 +18,153 @@ use crate::{
     state_ext::StateReadExt as _,
 };
 
+fn error_query_response(
+    err: Option<anyhow::Error>,
+    code: AbciErrorCode,
+    info: &str,
```

Comment:
IMO rename this as `msg` or something because the `info` field in the query response (which is the human readable `code`) and this `info` argument tripped me up

Reply:
Also to add to this: I think you can forego the `info` field entirely and lean on anyhow. If you encounter a `None`, you can either `None.context("something was not set").map_err(|e| error_to_query_response(e, <code>)`, or just `match thing_that_can_none() { None => ... }` as before.